CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ Changelog
 3.1.9 - 2023-06-16
 -------------------
 
+**Bug fix:**
```

Comment:
This is a past release. You need to add the bug fix under a new version entry `3.1.10 - 2023-XX-XX`.